CityOfDetroit/bloom
# fix: resolved visual bugs + override issues

# Pull Request Template

## Issue Overview

This PR addresses [#3048](https://github.com/bloom-housing/bloom/issues/3048)


## Description

This PR fixes the three issues mentioned below (screenshots in the linked issue):

1. Extra right margin on mid breakpoint 768-640
2. No focus state showing for modal button
3. Can we use the site standard for focus state on the image group focus state

## How Can This Be Tested/Reviewed?

1. Go to a listing with multiple images, resize the window to see that there is no extra padding on the right side
2. Tab through the page to see the corrected focus state on the image card
3. Click the multi-photo view, and tab through to see the focus state on the "Back to listing" button

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
